hibernate-core/src/test/java/org/hibernate/test/where/annotations/LazyManyToManyNonUniqueIdWhereTest.java

```diff
@@ -53,10 +53,15 @@ public void setup() {
 		doInHibernate(
 				this::sessionFactory, session -> {
 
-					session.createSQLQuery( "drop table MATERIAL_RATINGS" ).executeUpdate();
-					session.createSQLQuery( "drop table BUILDING_RATINGS" ).executeUpdate();
-					session.createSQLQuery( "drop table ASSOCIATION_TABLE" ).executeUpdate();
-					session.createSQLQuery( "drop table MAIN_TABLE" ).executeUpdate();
+					session.createSQLQuery("drop table if exists MATERIAL_RATINGS cascade").executeUpdate();
+					session.createSQLQuery("drop table if exists BUILDING_RATINGS cascade").executeUpdate();
+					session.createSQLQuery("drop table if exists ASSOCIATION_TABLE cascade").executeUpdate();
+					session.createSQLQuery("drop table if exists MAIN_TABLE cascade").executeUpdate();
```

Comment:
it would be consistent to retain the empty chars after `(` and before `)`.

Reply:
I agree -- I'll make this consistent.